Resources/Prototypes/DeltaV/Catalog/Fills/Lockers/heads.yml

```diff
@@ -82,6 +82,6 @@
     - id: BoxPDASecurity
     - id: WeaponEnergyGunMultiphase
     - id: HoSIDCard
-    - id: ClothingShoesBootsWinterHeadOfSecurity
```

Comment:
Shouldnt be removed, ideally resprite, some people want winter boots for aesthetics.

Reply:
If people want to resprite these they're welcome to.

Reply:
Hell, these are fine as is. Disagree with removal entirely.

Reply:
Feedback noted